documentation/api/io.strimzi.api.kafka.model.Rack.adoc

```diff
@@ -79,7 +79,10 @@ If the rack ID is not specified, or if it cannot find a replica with the same ra
 .Example showing client consuming from replicas in the same availability zone
 image::rack-config-availability-zones.png[consuming from replicas in the same availability zone]
 
-Consuming messages from the closest replicas can be used also in Kafka Connect for sink connectors which are consuming messages.
+Consuming messages from the closest replicas can be used also in Kafka Connect and Mirror Maker 2 for connectors which are consuming messages.
+Enabling rack awareness in the `KafkaConnect` and `KafkaMirrorMaker2` custom resource will not set any affinity rules, but you can also configure `affinity` or `topologySpreadConstraints`.
+For more information see xref:assembly-scheduling-str[].
```

Comment:
```suggestion
You can also configure Kafka Connect and MirrorMaker 2.0 so that connectors consume messages from the closest replicas.  
You enable rack awareness in the `KafkaConnect` and `KafkaMirrorMaker2` custom resources. 
The configuration does does not set affinity rules, but you can also configure `affinity` or `topologySpreadConstraints`.
For more information see xref:assembly-scheduling-str[].
```